test/controllers/comfy/blog/posts_controller_test.rb

```diff
@@ -75,10 +75,19 @@ def test_get_index_with_category_invalid
   end
 
   def test_get_show
+    expected_content =
+      "<h1>Default Title</h1>\n"\
+      "<p>\n"\
+      "Published on\n"\
+      ":\n"\
+      "#{@post.published_at.to_formatted_s(:short)}\n"\
+      "</p>\n"\
+      "blog post content\n"\
```

Comment:
I'd change this to `expected = <<~HTML` heredoc format.

Reply:
Fixture is also dynamic - `2.days.ago` so either I change the fixture, or keep it dynamic in the expected string.

Reply:
It appears `#published_at` is not used anywhere else, so I changed it to a fixed value